logdna/resource_key.go

```diff
@@ -0,0 +1,191 @@
+package logdna
+
+import (
```

Comment:
I think we should use 2 spaces instead of tab

Reply:
Golang prefers to use tabs for indentation. Using `go fmt` also goes to tabs. Some of the files in this repository do use spaces, but we should prefer to use tabs to be consistent with the go ecosystem IMO.